server/app/services/analytics_service.py

```
@@ -1629,34 +1678,48 @@ async def get_recent_calls_feedback(
+            start_date = datetime.now().date() - timedelta(days=30)
+        if end_date is None:
+            end_date = datetime.now().date()
+
+        # Import necessary SQLAlchemy functions
+        from sqlalchemy.orm import selectinload
+        from sqlalchemy import func
+
         # Build conditions for the query
         conditions = [
             Call.organization_id == organization_id,
             ServiceRecord.is_demo == False,
```

Ruff (0.13.3)

Avoid equality comparisons to `False`; use `not ServiceRecord.is_demo:` for false checks

Replace with `not ServiceRecord.is_demo`

(E712)